pkg/kapp/app/interfaces.go

```diff
@@ -24,7 +24,7 @@ type App interface {
 
 	CreateOrUpdate(map[string]string, bool) error
 	Exists() (bool, string, error)
-	Delete() error
+	Delete(checkResource bool) error
```

Comment:
We can keep it consistent with other functions.
```suggestion
	Delete(bool) error
```